### ISSUE TITLE
audio input level metering

### ISSUE THREAD

**Issue by Be-ing** [bigbluebutton/bigbluebutton#12384]
**Is your feature request related to a problem? Please describe.**
Speakers do not monitor their own voice while speaking. They have no awareness of how loud their voice is for the audience nor any hint they may be clipping. This results in different speakers being heard at very different volumes and some of them clipping.

**Describe the solution you'd like**
I think there should be a level meter on screen when the user has microphone input enabled. Additionally, I think there should be a notification advising the user to speak quieter when the audio input stays at a high level for more than a brief moment. It may be worth considering an inverse notification when the level is quiet, but that might get triggered too easily by background noise.

**Describe alternatives you've considered**
Zoom only shows a notification advising the user to turn down their microphone level when it is too loud. This does not help when the speaker is too quiet. Also, reducing the gain in software does not help if the signal has already been clipped in hardware before reaching the computer. Futhermore, explaining how to control the hardware input gain across the wide variety of software and hardware used is too complex of a topic for speakers to deal with during a meeting/conference. I think telling speakers to speak quieter or louder is much more practical advise.

**Comment by jibon57** [comment]
Similar: https://github.com/bigbluebutton/bigbluebutton/issues/12194

**Comment by Be-ing** [comment]
That is similar but insufficient. A compressor cannot fix a signal that is already clipped.

**Comment by Be-ing** [comment]
The need for this does not simply disappear with the passage of time.

**Comment by prlanzarin** [comment]
Note to self: verify whether this makes sense and associate with #14736

**Comment by Be-ing** [comment]
Adding a meter to the initial echo test is nice, but I think it should be visible throughout the duration of a meeting.

**Comment by prlanzarin** [comment]
Yeah, that's why I'm associating one to the other. The meter should be reusable.
